<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sel_exchange</class>
 <widget class="QWidget" name="sel_exchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择常用交易所</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>待选（双击选择）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>已选（双击移除）</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="lst_exchange"/>
     </item>
     <item>
      <widget class="QListWidget" name="lst_sels"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_sel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
